bazel/repository_locations.bzl

```diff
@@ -510,20 +510,36 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-07-28",
         cpe = "cpe:2.3:a:libevent_project:libevent:*",
     ),
+    net_colm_open_source_colm = dict(
+        project_name = "Colm",
+        project_desc = "The Colm Programming Language",
+        project_url = "https://www.colm.net/open-source/colm/",
+        version = "0.14.7",
+        sha256 = "6037b31c358dda6f580f7321f97a182144a8401c690b458fcae055c65501977d",
+        strip_prefix = "colm-{version}",
+        urls = ["https://www.colm.net/files/colm/colm-{version}.tar.gz"],
```

Comment:
Can we reference Github locations for these files? It's easier to reason about the dependency with canonical links to GH, including things like licenses.

Reply:
I may take time to try to switch to the GitHub repo. Previously, the repo and the tar ball contains different files. Ragel codes from the repo require Ragel itself to compile, which is ridiculous.

Reply:
Thanks @htuch. I came here to say the same thing. I've been running the following for months without issue
`        urls = ["https://github.com/adrian-thurston/ragel/archive/ragel-{version}.tar.gz"],`

Reply:
Yeah, I fixed that just now, thanks.